node/core/utils/types.py

```diff
@@ -9,6 +9,8 @@
 class Type(IntEnum):
     GENESIS = 0
     NODE_DECLARATION = 1
+    CONFIRMATION_VALIDATOR = 2
```

Comment:
these are block types, not node roles. you need another enum

Reply:
Added.